### ISSUE TITLE
Complete Multiple Tasks at Once

### ISSUE THREAD

**Issue by shcallaway** [shcallaway/todo#3]
I should be able to complete multiple tasks at once by providing multiple IDs. For example:

```
$ todo complete 123 456
Completed: Example task
Completed: Another example task
```

One question: How do you handle the case where one or more of the task IDs are invalid?

**Comment by shcallaway** [comment]
This also applies to removing multiple tasks at once.

**Comment by shcallaway** [comment]
Implemented in v1.1.
